frontend/src/scenes/experiments/ExperimentView/Overview.tsx

```diff
@@ -0,0 +1,93 @@
+import '../Experiment.scss'
+
+import { LemonDivider } from '@posthog/lemon-ui'
+import { useValues } from 'kea'
+import { getSeriesColor } from 'lib/colors'
+import { capitalizeFirstLetter } from 'lib/utils'
+
+import { InsightType } from '~/types'
+
+import { experimentLogic } from '../experimentLogic'
+
+export function Overview(): JSX.Element {
+    const {
+        experimentResults,
+        getIndexForVariant,
+        experimentInsightType,
+        sortedConversionRates,
+        highestProbabilityVariant,
+        areResultsSignificant,
+    } = useValues(experimentLogic)
+
+    function SignificanceText(): JSX.Element {
+        return (
+            <>
+                <span>Your results are&nbsp;</span>
+                <span className="font-semibold">{`${areResultsSignificant ? 'significant' : 'not significant'}`}.</span>
+            </>
+        )
+    }
+
+    if (experimentInsightType === InsightType.FUNNELS) {
+        const winningVariant = sortedConversionRates[0]
+        const secondBestVariant = sortedConversionRates[1]
+        const difference = winningVariant.conversionRate - secondBestVariant.conversionRate
+
+        return (
+            <div>
+                <h2 className="font-semibold text-lg">Summary</h2>
+                <div className="items-center inline-flex">
+                    <div
+                        className="w-2 h-2 rounded-full mr-1"
+                        // eslint-disable-next-line react/forbid-dom-props
+                        style={{ backgroundColor: getSeriesColor(winningVariant.index + 1) }}
+                    />
+                    <span className="font-semibold">{capitalizeFirstLetter(winningVariant.key)}</span>
+                    <span>&nbsp;is winning with a&nbsp;</span>
+                    <span className="font-semibold text-success items-center">{`${difference.toFixed(2)}% lift`}</span>
```

Comment:
<img width="687" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/6f37d810-7cf8-4e18-baea-66e991ad7a6a">

This is a bit confusing, since its lift in terms of percentage points, not an actual % increase, but is presented as a % increase.

Reply:
good catch, fixed!

<img width="820" alt="image" src="https://github.com/PostHog/posthog/assets/22996112/e4aeeef0-9151-4c32-bd3d-ef97de13a107">

Reply:
Thinking about this a bit more, I think I want a % increase here, seems like a better number to show? Since they can quite easily see the %points increase from the goal preview, but harder to calculate the % increase?

Reply:
Also reasonable argument, no strong feelings 👍